app/dts/bindings/behaviors/zmk,behavior-macro-press.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: Macro Press Mode Behavior
```

Comment:
Maybe people will just look at the docs website instead of the code for this info, but something more like "sets macro to press mode" would probably be more helpful.